cpp/include/cudf/functions.h

```diff
@@ -89,7 +89,8 @@ gdf_error gdf_column_view(gdf_column *column, void *data, gdf_valid_type *valid,
  * @param[in] valid Pointer to validity bitmask for the data.
  * @param[in] size Number of rows in the column.
  * @param[in] dtype Data type of the column.
- * @param[in] null_count The number of non-valid elements in the validity bitmask
+ * @param[in] null_count The number of non-valid elements in the validity bitmask.
+ * @param[in] extra_info see gdf_dtype_extra_info. Extra data for column description.
```

Comment:
For future reference, use @see to get doxygen to create an HTML link to the thing you reference.  Won't let it block this PR though.